exercism/bob/bob.src.js

```diff
@@ -0,0 +1,16 @@
+var Bob = function() {};
+
+  Bob.prototype.hey = function(what) {
```

Comment:
Be consistent in your code. In one case you have `if (` and in another you have `if(`. This may seem picky but its very important to code cleanliness.